NEWS

```diff
@@ -2,11 +2,144 @@ systemd System and Service Manager
 
 CHANGES WITH 232 in spe
 
+        * The new RemoveIPC= option can be used to remove IPC objects owned by
+          the user or group of a service when that service exits.
+
+        * Support for dynamically creating users for the lifetime of a service
+          has been added. If DynamicUser=yes is specified, a user and group IDs
+          will be allocated from the range 61184..65519 for the lifetime of the
+          service. They can be resolved using the new nss-systemd.so NSS
+          module. The module must be enabled in /etc/nsswitch.conf. Services
+          started in this way have PrivateTmp= and RemoveIPC= enabled, so that
+          any resources allocated by the service will be cleaned up when the
+          service exits.
+
+          nss-systemd module also always resolves root and nobody, making it
+          possible to have no /etc/passwd or /etc/group files in minimal
+          container systems.
```

Comment:
Hmm, most distros will require additional static GIDs. http://refspecs.linuxbase.org/LSB_4.1.0/LSB-Core-generic/LSB-Core-generic/usernames.html at least requires `bin` and `daemon` (although I'm not aware of anything actually using those), but e. g. `adm` is even being used by the journal. So I wonder if it is still premature to advertise this yet?